.idea/.name

```diff
@@ -1 +1 @@
-Caffeine Project
```

Comment:
That's a toss-up.  Doesn't hurt to have the .idea dir.  It means you can "Open Project" in the IDE vs import it, which is easier for external people to contribute to this library.

The downside is that Android Studio makes useless changes to those files, which end up in PRs.  We usually exclude .idea changes, but this PR was to update them.  

FWIW, our internal repos ignore `.idea`.